test/dist_autograd_test.py

```diff
@@ -77,6 +77,15 @@ def test_autograd_functions(self):
             send_functions = ctx._send_functions()
             self.assertEqual(1, len(send_functions))
 
+            # Ensure that the destination workerId is recorded on this context.
+            worker_ids = ctx._known_worker_ids()
+            self.assertEqual(len(worker_ids), 1)
+            self.assertEqual(dst_rank, worker_ids[0])
+            # TODO why not
```

Comment:
Instead of TODOs, create gh issues for follow ups. This way we can easily track this. Although, in this particular case, what you've done here sounds good. Why do we need to use `rpc.get_worker_info`?

Reply:
I thought accessing the worker id through the worker info struct would be a bit clearer, but yeah, I agree this seems fine.